feature-libs/customer-ticketing/styles/components/_customer-ticketing-list.scss

```diff
@@ -138,10 +149,12 @@
   }
 
   .cx-text-green {
+    text-decoration: none;
```

Comment:
Will only flag this, but applies to any hovered table data. 

When hovering on the table data, it is normal for our table to show an underline, so please remove.